internal/controller/database_secret_handler.go

```diff
@@ -132,15 +132,15 @@ func (e *secretEventHandler) Update(ctx context.Context, evt event.UpdateEvent,
 	}
 }
 
-func (e *secretEventHandler) Delete(context.Context, event.DeleteEvent, workqueue.RateLimitingInterface) {
+func (e *secretEventHandler) Delete(ctx context.Context, evt event.DeleteEvent, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
```

Comment:
I don't think `ctx` is used in the function, so why not keeping it as an unnamed import?

Reply:
ok